mailpoet/assets/js/src/newsletters/send.tsx

```diff
@@ -764,6 +774,13 @@ class NewsletterSendComponent extends Component<
   updatePendingApprovalState = (mssKeyPendingApproval: boolean): void =>
     this.setState({ mssKeyPendingApproval });
 
+  saveDraftNewsletter = (afterSaveCallback: () => void) => {
```

Comment:
Could this be an async function instead? Passing callback seems a bit old-school, but maybe we're using old code here, that is not easy to wrap with a promise...?

Reply:
Thanks for pointing this out! I updated this in ee7e9fee7a7bd971c888cb57d70aafdff438c7a8. Let me know if it looks off in any way.